node/public/script/main.js

```diff
@@ -33,14 +33,14 @@ if (window.location.pathname == "/") {
 
     //on attend le signal init du serveur qui envoie la main et le board
     socket.on('init', function (init) {
-
+        console.log(init.board);
         //on parcours le board pour l'afficher
         for (indexBoard = 0; indexBoard < 4; indexBoard++) {
```

Comment:
`indexBoard` and `indexBoard2` are badly named variables. When doing 2 for loops, you can usually name `rows` and `columns`.

Reply:
In javascript, and in functional programming in general, you would rather use the [forEach method](https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/).

Could be something like:

```
board.forEach((row) => {
    row.forEach((card) => {
        // do something with the card
    });
});
```